src/printer.js

```diff
@@ -1447,29 +1447,30 @@ function genericPrintNoParens(path, options, print, args) {
 
       return concat(parts);
     case "ClassProperty":
+    case "TSAbstractClassProperty":
       if (n.static) parts.push("static ");
 
-      var key;
+      var key = [];
+
+      var variance = getFlowVariance(n, options);
+
+      if (variance) {
+        key.push(variance);
+      } else if (n.accessibility) {
```

Comment:
I don't think `n.accessibility` is ever set in flow, so it should be ok to break it out. Tests still pass.